src/e2e-test/regression-test/util/interceptors.js

```diff
@@ -1,72 +1,96 @@
-// -------------------------------------------------------------------------------------------------
```

Comment:
Can you check the line endings in these files?

Reply:
Yes and I'm just checking it now. "git config --global core.autocrlf true" it set many times, may be history issues.